### DISCUSSION TITLE
Add linter option to disable `all` rules

### DISCUSSION THREAD

**Discussion by jake8655** [biomejs/biome#2822] [Ideas]
## Description
Currently the `linter.rules.all` option accepts a boolean:
- `true` -> enable all rules (except nursery)
- `false` -> _don't enable_ all rules (default)

It would be nice to instead have the option to **enable** or **disable** all rules as such:
- `"on"` -> enable all rules (except nursery)
- `"off"` -> _disable_ all rules
- `not set` -> default (same as false in the current version)

## Use case
You want to disable all rules except some that you manually whitelist by setting them to `"warn"` or `"error"`.

**Comment by ematipico** [comment]
You can achieve what you want by disabling the recommended rules. Is that correct?

**Reply by jake8655** [reply]
@ematipico Yes! Thank you for pointing this out, I didn't know that.

In that case, there's no need for my proposed changes. Maybe there could be a small note about this somewhere in the docs though, as it doesn't seem to be that intuitive at first. What are your thoughts on that?

**Reply by ematipico** [reply]
We welcome any change that could help making things clear

**Reply by jake8655** [reply]
Alright, I will create a PR shortly then.

Thank you for your help!
